libs/dh/esett/feature-balance-responsible/src/lib/dh-balance-responsible.component.html

```diff
@@ -14,39 +14,99 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<watt-card vater inset="ml" *transloco="let t; read: 'eSett.balanceResponsible'">
-  <vater-flex fill="vertical" gap="m">
-    <vater-stack direction="row" gap="s">
-      <h3>{{ t("tabHeading") }}</h3>
-      <span class="watt-chip-label">{{ totalCount }}</span>
-
-      <vater-spacer />
+<watt-data-table
+  *transloco="let t; read: 'eSett.balanceResponsible'"
+  vater
+  inset="ml"
+  [enableSearch]="false"
+  [error]="dataSource.error"
+  [ready]="dataSource.called"
+  #wattDataTable
+>
+  <h3>{{ t("tabHeading") }}</h3>
 
+  <watt-data-actions>
+    <vater-stack direction="row" gap="s">
       <watt-button
         icon="download"
         variant="text"
-        [loading]="isDownloading"
-        (click)="download(url)"
-        >{{ "shared.download" | transloco }}</watt-button
-      >
+        [loading]="isDownloading()"
+        (click)="download(url())"
+        >{{ "shared.download" | transloco }}
+      </watt-button>
 
-      <dh-balance-responsible-importer />
+      <dh-balance-responsible-importer [uploadUrl]="importUrl()" />
     </vater-stack>
+  </watt-data-actions>
+
+  <watt-table
+    *transloco="let resolveHeader; read: 'eSett.balanceResponsible.columns'"
+    [dataSource]="dataSource"
+    [columns]="columns"
+    [resolveHeader]="resolveHeader"
+    [loading]="dataSource.loading"
+    [activeRow]="selection()"
+    (rowClick)="navigation.navigate('details', $event.id)"
+  >
+    <ng-container *wattTableCell="columns['received']; header: t('columns.received'); let message">
+      {{ message.receivedDateTime | wattDate: "long" }}
+    </ng-container>
```

Comment:
nitpick: both `resolveHeader` and `header` property are set. Let's use only one.